app/user/rest.py

```diff
@@ -230,7 +230,14 @@ def verify_user_password(user_id):
         return jsonify({}), 204
     else:
         increment_failed_login_count(user_to_verify)
-        message = "Incorrect password"
+        # We want to get the user again to check the failed login count
+        user_to_verify = get_user_by_id(user_id=user_id)
```

Comment:
Why bother going back to the DB for the user again?

Reply:
Right but I think the object is updated in place so I don't think we need to bother with the extra db call!

Reply:
let me look at it again, thanks for pointing it out